pagination.js

```diff
@@ -0,0 +1,68 @@
+// Creating Pagination.
+var currentPage = 1;
+function apply_pagination(paginList, data) {
+    paginList.innerHTML = "";
+    let prevBtn = document.createElement('button');
```

Comment:
inside this function i can see redundant codes  , please write a common logic and call that function

Reply:
line 5 to 8 and 23 to 26